modules/distribution/src/repository/resources/dbscripts/apimgt/oracle_rac.sql

```diff
@@ -87,9 +88,10 @@ CREATE INDEX IDX_AT_CK_AU ON IDN_OAUTH2_ACCESS_TOKEN(CONSUMER_KEY_ID, AUTHZ_USER
 CREATE INDEX IDX_TC ON IDN_OAUTH2_ACCESS_TOKEN(TIME_CREATED)
 /
 CREATE TABLE IDN_OAUTH2_AUTHORIZATION_CODE (
+            CODE_ID VARCHAR (255),
             AUTHORIZATION_CODE VARCHAR2 (512),
             CONSUMER_KEY_ID INTEGER,
-              CALLBACK_URL VARCHAR2 (1024),
+	          CALLBACK_URL VARCHAR2 (1024),
```

Comment:
Directly copied content from https://github.com/wso2/carbon-apimgt/tree/v5.0.4/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql as apim version upgraded to v5.04